packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -68,6 +68,12 @@ export interface IPushOptions {
     };
     fcmOptions?: { analyticsLabel?: string };
   };
+  subscriber: object;
+  step: {
+    digest: boolean;
+    events: object[] | undefined;
+    total_count: number | undefined;
+  };
```

Comment:
Can't here we reuse `IPushOptions['step']`? If it is needed let's move it to `@novu/shared`.

Reply:
this is the `IPushOptions` interface :P

Reply:
You got my point 😂